build.gradle

```diff
@@ -51,8 +51,12 @@ task cleanTarget(type: MavenExec) {
     args 'clean'
 }
 
+task assemble() {
+}
+
 // TODO define inputs and outputs
```

Comment:
don't know what that means (might be due to a lack of understanding of Gradle, though…)

Reply:
had to include this empty task to satisfy travis (that sees a gradle build configuration and assumes that this task must be executed for a java project)